### ISSUE TITLE
Manual replaceOnChanges:["template.templateName"] required when SES email template name changed of pulumi resource

### ISSUE THREAD

**Issue by Jimmy89** [pulumi/pulumi-aws-native#834]
### What happened?

Can be added to the list of https://github.com/pulumi/pulumi-aws-native/issues/736.

For the `aws.ses.Template` resource the `template: { templateName: "NAME" }` option must trigger a replacement if the `NAME` changed. However, it does not and therefore triggers a FailedUpdate on the resource 

### Expected Behavior

A replacement is triggered when the template name is adjusted.

### Steps to reproduce

Create a resource as:

```
const reminderEveryWeekTemplate = new aws.ses.Template("weekly-reminder-template", {template: {
  templateName: "SomeName",
  subjectPart: "SUBJECT",
  textPart: "Hello",
  htmlPart: "<b>Hello</b>",
}});
```` 

Now change `SomeName` into `SomeOtherName`. Run it again.

### Output of `pulumi about`

```
Dependencies:
NAME                VERSION
@pulumi/aws-native  0.52.0
@pulumi/pulumi      3.57.1
@pulumi/random      4.11.2
```

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by danielrbradley** [comment]
Thanks @Jimmy89 have added to the list on #736 👍

**Comment by mjeffryes** [comment]
This should be resolved by #1053; changes to templateName will result in a replace rather than an update.
